XTS/wsat-jta-multi_hop/pom.xml

```diff
@@ -219,22 +220,22 @@
             <build>
                 <plugins>
                     <plugin>
-                        <groupId>org.apache.maven.plugins</groupId>
                         <artifactId>maven-surefire-plugin</artifactId>
                         <configuration>
                             <skip>false</skip>
+                            <argLine>${ipv6.server.jvm.args}</argLine>
```

Comment:
Because otherwise it would not reflect the same settings as our other IPv6 poms...